### PR TITLE
refactor: get rid of Gp prefix

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Context: https://github.com/cowprotocol/cowswap/pull/4496#discussion_r1627714263

Removed `Gp*` prefix from entities names.

# To Test

Can be skipped, the refactoring is done by auto replacement with IDEA

### CODE REVIEW

File: apps/cowswap-frontend/src/api/cowProtocol/priceApi.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { RAW_CODE_LINK } from '@cowprotocol/common-const'
 import { environmentName } from '@cowprotocol/common-utils'
 import { SupportedChainId, mapSupportedNetworks } from '@cowprotocol/cow-sdk'
 
-import { GpPriceStrategy } from 'legacy/state/gas/atoms'
+import { PriceStrategy as PriceStrategyEnum } from 'legacy/state/gas/atoms'
```
Comment (shoom3301) [inline apps/cowswap-frontend/src/api/cowProtocol/priceApi.ts:5]: The only manual change due to names conflict

File: libs/common-const/src/common.ts [hunk 2]
```diff
@@ -58,10 +48,6 @@ export const COWSWAP_ETHFLOW_CONTRACT_ADDRESS: Record<Env, Record<SupportedChain
   barn: mapSupportedNetworks((chain) => EthFlowBarn[chain].address),
 }
 
-export const GP_SETTLEMENT_CONTRACT_ADDRESS = COW_PROTOCOL_SETTLEMENT_CONTRACT_ADDRESS
```
Comment (shoom3301) [inline libs/common-const/src/common.ts:61]: Replaced those consts with consts from SDK

**Review by anxolin — APPROVED** [review]
bye bye 🦖 code

File: apps/cowswap-frontend/src/legacy/hooks/useRefetchPriceCallback.tsx [hunk 3]
```diff
@@ -21,12 +21,12 @@ import { getBestQuote, getFastQuote, QuoteResult } from 'legacy/utils/price'
...
-import GpQuoteError, {
-  GpQuoteErrorCodes,
-  GpQuoteErrorDetails,
+import { ApiErrorCodes, isValidOperatorError } from '../../api/cowProtocol/errors/OperatorError'
+import QuoteApiError, {
+  QuoteApiErrorCodes,
+  QuoteApiErrorDetails,
   isValidQuoteError,
-} from 'api/gnosisProtocol/errors/QuoteError'
+} from '../../api/cowProtocol/errors/QuoteError'
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/legacy/hooks/useRefetchPriceCallback.tsx:29]: Here and more places with the relative import to fix

**Review by alfetopito — COMMENTED** [review]
A few naming suggestions and many places where the imports are changing from absolute to relative. Did not mark them all.

**Review by alfetopito — APPROVED** [review]

**Pull request merged**